tracer/src/Datadog.Trace/AppSec/Security.cs

```diff
@@ -359,6 +362,22 @@ private void InstrumentationGatewayInstrumentationGatewayEvent(object sender, In
             }
         }
 
+        private void ReportWafInitInfoOnce(object sender, InstrumentationGatewaySecurityEventArgs e)
```

Comment:
Am I understanding correctly that this is going to be sent on only one span for the entire application lifetime? This seems like a great use for telemetry, though it's currently opt-in for all environments except Azure App Service

Reply:
yes, indeed, it was supposed to be in telemetry but as it's not available everywhere, decision should be revisited once it is